test/index.test.js

```diff
@@ -106,27 +109,44 @@ describe('nexus', () => {
     })
   }
 
+  function testStringPresentInModule(stringExportName, module) {
+    it(`exposes ${stringExportName}`, () => {
+      expect(typeof module[stringExportName]).toBe('string')
+    })
+  }
+
   function assertValueFunctionOrObject(value) {
     // this check provides some safety on the MODULE_STRUCTURE above so it is not extended
     // with something unexpected unintentionally
-    if (value !== 'function' && typeof value !== 'object') {
-      throw new Error(`${value} must be string 'function' or an object`)
+    if (value !== 'function' && value !== 'string' && typeof value !== 'object') {
+      throw new Error(`${value} must be string 'function', 'string' or an object`)
     }
   }
 
   function testExportAgainstNexus(exportName, exportValue, path) {
     assertValueFunctionOrObject(exportValue)
 
-    exportValue === 'function' // eslint-disable-line
-      ? testFunctionPresentInModule(exportName, eval(path.join('.'))) // eslint-disable-line
-      : testModuleStructure(exportValue, [ ...path, exportName ]) // eslint-disable-line
+    switch (exportValue) {
```

Comment:
This is an artefact from export th event names for pusher from nexus; they are notw exported on the `OspinPusherClient`. I still would like to keep it.

Reply:
noice extension - so this allows strings to be exported now?

Reply:
yes